### PR TITLE
Feature/add databricks workflows

### PR DESCRIPTION
**Pull request by vitoravancini**
Add databricks worflow models

### CODE REVIEW

File: models/staging/adf_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql [hunk 1]
```diff
@@ -45,14 +45,16 @@ pipeline_dedup as (
...
+        ,pipelines.name as dag_name
         ,triggers.*
     from pipeline_dedup pipelines
     left join triggers_renamed triggers
     on pipelines.name = triggers.pipeline_name
 )
 select 
-    dag_id
+    dag_name as dag_id
+    ,dag_name as dag_name
```
Comment (jessicaoblira) [inline models/staging/adf_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql:57]: Pig @vitoravancini , on line 56 and 57, why did you rename these columns? I believe you just need to pull the columns (dag_id and dag_name) from the cte above. 
And also add a space after the comma :)

File: models/staging/airflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql [hunk 2]
```diff
@@ -1,7 +1,8 @@
 with
     renamed as (
         select distinct
-            dag_id
+            dag_id as dag_id
+            , dag_id as dag_name
```
Comment (jessicaoblira) [inline models/staging/airflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql:5]: line 5: Should it be dag_id as dag_name? If you haven't changed the column name, why rename it? Just select the columns for the model.

File: models/staging/databricks_workflow_sources/source.yml [hunk 3]
```diff
@@ -0,0 +1,121 @@
+version: 2
+
+sources:
+  - name: raw_databricks_workflow_monitoring
+    description: "Dados brutos retirados do databricks para análise de monitoramento do databricks."
+    schema: dev_vitor_avancini
```
Comment (jessicaoblira) [inline models/staging/databricks_workflow_sources/source.yml:6]: Change the schema name ;)

File: models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql [hunk 4]
```diff
@@ -0,0 +1,23 @@
+with renamed as (
+    select 
+        job_id as dag_id,
+        settings.name as dag_name
+        , "not_implemented_for_databricks_workflow" as dag_description
```
Comment (jessicaoblira) [inline models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql:5]: Put the commas at the beginning of the subsequent lines

File: models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag_run.sql [hunk 5]
```diff
@@ -0,0 +1,16 @@
+with renamed as (
+    select
+        run_id as dag_run_id
+        , job_id as dag_id
+        , from_unixtime(start_time / 1000) run_date
+        , state.result_state as dag_state
+        , `trigger`as external_trigger
+        , from_unixtime(start_time / 1000) as execution_start_date
+        , from_unixtime(end_time / 1000) execution_end_date
```
Comment (jessicaoblira) [inline models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag_run.sql:9]: Please, add the 'as' keyword when aliasing columns

File: models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_task_instance.sql [hunk 6]
```diff
@@ -0,0 +1,32 @@
+with tasks as (
+    select 
+        *
+    from 
+        {{ source('raw_databricks_workflow_monitoring', 'databricks_job_runs') }} dag_runs
+    lateral view explode(dag_runs.tasks) as task
+), renamed as (
```
Comment (jessicaoblira) [inline models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_task_instance.sql:7]: Please, add a indentation here (line 7)

File: models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_task_instance.sql [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        , case 
+            when task.notebook_task.notebook_path is not null then task.notebook_task.notebook_path 
+            else task.task_key
+        end as operator
+        , 'not_implemented_for_databricks_workflow' as map_index
+    from tasks
+)
+select 
+    {{ dbt_utils.surrogate_key(['task_id', 'dag_id', 'run_id']) }} as task_instance_sk,
+    *
```
Comment (jessicaoblira) [inline models/staging/databricks_workflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_task_instance.sql:30]: Put the commas at the beginning of the subsequent lines
like 
{{ dbt_utils.surrogate_key(['task_id', 'dag_id', 'run_id']) }} as task_instance_sk
, *

**Review by jessicaoblira — COMMENTED** [review]
Pig, I added some small comments throughout the PR, but just to ensure the indicium's codestyle :) great job!!

File: models/staging/adf_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql (hunk 1 above)
Comment (vitoravancini) [reply models/staging/adf_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql:57]: trying to mantain the pattern for always having 'as', which I just saw it is not really a pattern.
Removed the uncessary 'as'
fixed the comma

File: models/staging/airflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql (hunk 2 above)
Comment (vitoravancini) [reply models/staging/airflow_sources/stg_airflow_monitoring_raw_airflow_monitoring_dag.sql:5]: removed unecessary as

File: models/staging/databricks_workflow_sources/source.yml (hunk 3 above)
Comment (vitoravancini) [reply models/staging/databricks_workflow_sources/source.yml:6]: :(

**Review by jessicaoblira — APPROVED** [review]
